src/utils/endpoints/accuracyEndpoints.ts

```diff
@@ -0,0 +1,3 @@
+export const accuracyStatisticsEndpoint = (apiUrl: string): string => {
+  return `${apiUrl}/statistics`
+}
```

Comment:
I would structure differently the endpoints file. I don't really think there is going to be more accuracy endpoints, so instead of having an accuracyStatisticsEndopoint file, let's just create an pdrBackendEndpoints file, and there we can include all the paths that are going to be based on the apiUrl provided by the pdrBackend project